frontend/src/Indexer/Add/AddIndexerModalContent.js

```diff
@@ -30,6 +30,12 @@ const columns = [
     isSortable: true,
     isVisible: true
   },
+  {
+    name: 'language',
+    label: 'Language',
```

Comment:
I see other things in here needs translated as well.. Can you possibly do the missing translations in this file since you're in here?

Reply:
This should be resolved now